src/modules/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.22)
+
+project(ModuleManager)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_subdirectory(inventory)
+
+add_library(ModuleManager src/moduleManager.cpp)
```

Comment:
Check also the usage of `configure_target` here (`src/cmake/ConfigureTarget.cmake`).